### PR TITLE
Initial configuration of (unit) tests for helm release to be added under OpenShift CI

### PR DESCRIPTION
**Pull request by ldimaggi**
…

### CODE REVIEW

File: ci-operator/config/redhat-developer/helm/OWNERS [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+approvers:
+- otaviof
+- sbose78
+options: {}
+reviewers:
+- akashshinde
+- akram
+- avni-sharma
+- baijum
+- bkundu
```
Comment (pmacik) [inline ci-operator/config/redhat-developer/helm/OWNERS]: @ldimaggi I believe Bama's github account is @bamachrn, isn't it?

Comment (ldimaggi) [reply ci-operator/config/redhat-developer/helm/OWNERS]: Thanks!

**Comment by ldimaggi** [comment]
This OpenShift CI job pull request can be merged: https://github.com/openshift/release/pull/5685

Only (2) checks in the pull request are failing now: the unit test job is running to completion, but some of the individual tests are failing, and the pj-rehearse job is failing because the unit test job is failing.

Thanks to Pavel, Baiju, and Bama for their assistance in getting this set up!

**Pull request merged**